server/config.py

```diff
@@ -6,3 +6,9 @@
 SQLALCHEMY_DATABASE_URI = f'postgresql://{DB_USER}:{DB_PASSWORD}@localhost/crm_test'
 SECRET_KEY = os.environ.get('SECRET_KEY')
 BCRYPT_LOG_ROUNDS = 12
+
+# The environment variables need to be available or nothing will work so throw an error if they're not set
+assert SECRET_KEY
+assert DB_USER
+assert DB_PASSWORD
+assert SQLALCHEMY_DATABASE_URI
```

Comment:
I like this

Just so you know, in the past I have also attained this result by doing:
SECRET_KEY = os.environ.['SECRET_KEY'] instead of SECRET_KEY = os.environ.get('SECRET_KEY'), as it'll throw an error

I like this you have done because it's clear to future developers that you need these variables